src/components/lesson-item/lesson-item.module.scss

```diff
@@ -0,0 +1,29 @@
+@import '@styles/theme.module.scss';
+
+.root {
+    display: grid;
```

Comment:
couldn't it just be flex?
with a direction row for desktop and a column for mobile?

Reply:
I want the fields aligned, because if I use `flex` the prices for example are dependent on the title width.

Reply:
you can give the children flex: 1 and then they are same width
just a suggestion